[2024-07-07T20:36:58.435+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T20:36:58.450+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl backfill__2019-01-01T00:00:00+00:00 [queued]>
[2024-07-07T20:36:58.457+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl backfill__2019-01-01T00:00:00+00:00 [queued]>
[2024-07-07T20:36:58.457+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-07-07T20:36:58.467+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2019-01-01 00:00:00+00:00
[2024-07-07T20:36:58.473+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=235) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T20:36:58.474+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'backfill__2019-01-01T00:00:00+00:00', '--job-id', '515', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpve_5kg8h']
[2024-07-07T20:36:58.475+0000] {standard_task_runner.py:63} INFO - Started process 238 to run task
[2024-07-07T20:36:58.475+0000] {standard_task_runner.py:91} INFO - Job 515: Subtask games_etl
[2024-07-07T20:36:58.513+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl backfill__2019-01-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-07T20:36:58.572+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='backfill__2019-01-01T00:00:00+00:00'
[2024-07-07T20:36:58.573+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T20:36:58.574+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-07T20:36:58.823+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-07T20:36:58.823+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T20:36:58.830+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=backfill__2019-01-01T00:00:00+00:00, execution_date=20190101T000000, start_date=20240707T203658, end_date=20240707T203658
[2024-07-07T20:36:58.849+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T20:36:58.862+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-07T20:36:58.863+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
